src-srv/utils/CollaborationServer.ts

```diff
@@ -205,7 +206,7 @@ export class CollaborationServer {
    */
   async #fetchDocument({ documentName: uuid, document: yDoc, context }: fetchPayload): Promise<Uint8Array | null> {
     // Init tracking document. Must not be fetched from cache when starting up the server fresh.
-    if (uuid === 'document-tracker' && !this.#openDocuments) {
+    if (uuid === 'document-tracker') {
```

Comment:
I think we need to discuss this logic a bit more. This will reset the local #openDocuments every time a new user fetches the _document-tracker_ document. It might be that we need to handle this special document the same as other documents and allow it to be fetched from cache...

Reply:
Yes, this needs to be discussed. Previous solution led to #fetchDocument trying to open `document-tracker` from repository, which threw errors. (unsure if that was just in dev server)